File: web/desktop/users-app/Windows/DatabaseConfig.cs
// Same database connection
using MySql.Data.MySqlClient;
public class Database {
    private string connString = "Server=localhost;Database=tigerex;Uid=root;Pwd=pass;";
    // Connects to shared MySQL backend
    public MySqlConnection GetConnection() {
        return new MySqlConnection(connString);
    }
}

File: web/android/users-app/Kotlin/DatabaseConfig.kt
object DatabaseConfig {
    const val DB_URL = "jdbc:mysql://localhost:3306/tigerex"
    const val USER = "tigerex_user"
    const val PASS = "password"
    // Same database connection
    fun getConnection(): Connection = DriverManager.getConnection(DB_URL, USER, PASS)
}
